org.eclipse.lsp4xml.extensions.maven/src/test/java/org/eclipse/lsp4xml/extensions/maven/test/ExtensionTest.java

```diff
@@ -62,6 +63,32 @@ public void tearDown() {
 		assertTrue(completion.getRight().getItems().stream().map(CompletionItem::getLabel).anyMatch("runtime"::equals));
 	}
 
+	@Test public void testVersionCompletion() throws IOException, InterruptedException, ExecutionException, URISyntaxException {
+		TextDocumentItem textDocumentItem = createTextDocumentItem("/pom-version-complete.xml");
+		DidOpenTextDocumentParams params = new DidOpenTextDocumentParams(textDocumentItem);
+		connection.languageServer.getTextDocumentService().didOpen(params);
+		Either<List<CompletionItem>, CompletionList> completion = connection.languageServer.getTextDocumentService().completion(new CompletionParams(new TextDocumentIdentifier(textDocumentItem.getUri()), new Position(13, 13))).get();
+		while (completion.getRight().getItems().stream().map(CompletionItem::getLabel).anyMatch(label -> label.contains("Updating Maven repository index...")) || !completion.getRight().getItems().stream().map(CompletionItem::getLabel).anyMatch(label -> label.contains("3.6.3")) ) {
```

Comment:
Not the most readable while loop condition.
What's happening here is the maven central index takes some time to download asynchronously, so we keep retriggering the completion to see if it's still computing, or the result appeared.

I just realized we should also check for error messages in the while loop conditon... needs to be added before merge.